circle.yml

```diff
@@ -0,0 +1,24 @@
+---
+version: 2
+jobs:
+  build:
+    working_directory: /go/src/github.com/resin-io/sshproxy
+    docker:
+      - image: golang:1.8.3-stretch
```

Comment:
Even better, it'd be nice then to have a brief note in the README on what the local build workflow is.